mmcls/models/utils/attention.py

```diff
@@ -513,3 +514,89 @@ def forward(self, x):
         x = self.proj(x)
         x = self.proj_drop(x)
         return x
+
+
+class TinyViTAttention(BaseModule):
+    """TinyViT Attention.
```

Comment:
Better to describe what's the difference between this module and the usual attention module.